### PR TITLE
Add windows nodeSelectors and tolerations to rancher-monitoring

### PR DESCRIPTION
**Pull request by aiyengar2**
Related Issue: https://github.com/rancher/rancher/issues/28327, https://github.com/rancher/rancher/issues/31498, https://github.com/rancher/rancher/issues/31497

### CODE REVIEW

**Comment by aiyengar2** [comment]
@brendarearden I modified the purpose of this PR to just add nodeSelectors + tolerations to Monitoring and moved adding rancher-windows-exporter to its own PR.

This PR is now effectively a continuation of https://github.com/rancher/charts/pull/998, which modified rancher-pushprox to work in a Windows cluster. Now, the remaining components in rancher-monitoring have also been modified for this.

**Review by nickgerace — COMMENTED** [review]
Where are `linux-node-selector` and `linux-node-tolerations` defined?

File: packages/rancher-monitoring/generated-changes/patch/templates/_helpers.tpl.patch [hunk 1]
```diff
@@ -68,6 +68,28 @@
...
 +{{- end }}
 +
++# Windows Support
++
++{{/*
++Windows cluster will add default taint for linux nodes,
++add below linux tolerations to workloads could be scheduled to those linux nodes
++*/}}
++
++{{- define "linux-node-tolerations" -}}
```
Comment (aiyengar2) [inline packages/rancher-monitoring/generated-changes/patch/templates/_helpers.tpl.patch:78]: define tolerations

Comment (nickgerace) [reply packages/rancher-monitoring/generated-changes/patch/templates/_helpers.tpl.patch:78]: AH it was the helpers. ty

**Review by nickgerace — APPROVED** [review]

**Review by brendarearden — APPROVED** [review]
LGTM

**Pull request merged**